submission/notifications/src/main/kotlin/ebi/ac/uk/notifications/api/RtClient.kt

```diff
@@ -0,0 +1,43 @@
+package ebi.ac.uk.notifications.api
+
+import arrow.core.getOrElse
+import ebi.ac.uk.notifications.exception.InvalidResponseException
+import ebi.ac.uk.notifications.exception.InvalidTicketIdException
+import ebi.ac.uk.notifications.integration.RtConfig
+import ebi.ac.uk.util.collections.second
+import ebi.ac.uk.util.regex.match
+import ebi.ac.uk.util.regex.secondGroup
+import org.springframework.util.LinkedMultiValueMap
+import org.springframework.web.client.RestTemplate
+import org.springframework.web.client.postForEntity
+import org.springframework.web.util.UriComponentsBuilder
+
+class RtClient(private val rtConfig: RtConfig) {
+    fun createTicket(content: String): String {
```

Comment:
The idea of creating a client is to encapsulate rtConfig by chequing [documentation](https://rt-wiki.bestpractical.com/wiki/REST#Ticket_Create)  I see that values are passed in string with line break separated, making a client receiving this string and sending the same string does not simplify RT. What I  would expect is client receiving a  RtNotification class which contains subject email and textContent and like that so internally we used it.